src/V3Stats.cpp

```diff
@@ -34,8 +34,18 @@ class StatsVisitor final : public VNVisitor {
 private:
     // NODE STATE/TYPES
 
+    // Input:
+    //  AstVar::user1 // Counts assignments to a signal, per instance
+    //  AstVar::user2 // Source line of potential 'RAM' assignment
+    VNUser1InUse m_inuser1;
+    VNUser2InUse m_inuser2;
+
     using NameMap = std::map<const std::string, int>;  // Number of times a name appears
 
+    typedef std::vector<AstVarRef*> VarVec;
+
+    const int minRAMBits = 8192;  // Arbitary threshold to prevent lots of tiny RAMs being reported
```

Comment:
Perhaps better as an option that can be overridden, but we can leave that for later.